### PR TITLE
add `mac_gsize` for danRer10

### PR DESCRIPTION
**Pull request by mashehu**
I added the missing `mac_gsize` for danRer10. Is that something I need to add to the changelog?

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] `CHANGELOG.md` is updated

### CODE REVIEW

**Comment by drpatelh** [comment]
Yes, please can you add that to the CHANGELOG 👍 I thought we were automatically checking that this was updated. Ah, we havent ported that test to GitHub actions yet. 

Just out of interest how did you decide on this number?

**Comment by mashehu** [comment]
The number is based on this publication: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5562042/, with a similar number reported at other places. Should I cite it in the changelog?

**Comment by drpatelh** [comment]
Perfect place for it, yes 👍 Thanks.

**Comment by mashehu** [comment]
I checked again and it was actually for the previous genome version. I updated it now slightly based on https://biostar.galaxyproject.org/p/18272/.

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 * Added CI test to check for PRs against `master` in tools repo
 * CI PR branch tests fixed & now automatically add a comment on the PR if failing, explaining what is wrong
 * Describe alternative installation method via conda with `conda env create`
+* Added `mac_gsize` for danRer10, based on https://biostar.galaxyproject.org/p/18272/
```
Comment (drpatelh) [inline CHANGELOG.md]:
```suggestion
* Added `macs_gsize` for danRer10, based on [this post](https://biostar.galaxyproject.org/p/18272/)
```

**Pull request merged**